docs/Tutorial-Dataloader.ipynb

```diff
@@ -105,7 +105,7 @@
     "repo.init(user_name=username, user_email=email, remove_old=True)\n",
     "co = repo.checkout(write=True)\n",
     "\n",
-    "images_aset = co.arraysets.init_arrayset('images', shape=img_shape, dtype=np.uint8, named_samples=False)\n",
+    "images_aset = co.columns.init_arrayset('images', shape=img_shape, dtype=np.uint8, named_samples=False)\n",
```

Comment:
It's probably "WIP". Just making  sure that you didn't miss this one out

Reply:
Got it!